sportnet/sportnet/playgrounds/tests/test_models.py

```diff
@@ -59,3 +35,38 @@ def test_delete(self):
         self.assertEqual(len(models.Sport.objects.all()), 2)
         models.Sport.objects.all().first().delete()
         self.assertEqual(len(models.Sport.objects.all()), 1)
+
+
+class PlaceEventTest(TestCase):
+
+    def setUp(self):
+        call_command('flush', '--noinput')
+        self.first_test = PlaceEvent(city='Moscow', street='Baumanskaya')
```

Comment:
```suggestion
        self.place = PlaceEvent(city='Moscow', street='Baumanskaya')
```
Старайся именовать переменные так, чтобы из названия было понятно, что это такое.
`first_test` например точно не подходит т.к. эта переменная не содержит никакого действия или что то в этом роде, и вовсе тестом не является. Это просто объект местоположения и не более.